lib/msal-node/package.json

```diff
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "@azure/msal-common": "^9.0.1",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.0",
```

Comment:
When are you planning on making a release with this change for @azure/msal-node?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5473

Reply:
This will be released on Monday, January 9th. Thanks for your patience!

Reply:
Yes. We release on the first Monday of every month, since yesterday was a holiday we've pushed it to the 9th. Ignore the release PR name, it's automated.

Reply:
hello, @tnorling. I do not see any new release on the npm registry today. There is a new planned date for publishing the new version? Thank you very much.

Reply:
@franher we'll be starting the release in a couple of hours :)